language/stdlib/transaction_scripts/ol_autopay_disable.move

```diff
@@ -1,12 +1,12 @@
 script {
-    use 0x1::AutoPay;
```

Comment:
We will probably want another transaction that users can send which migrates, or otherwise, erases the original Autopay.